packages/fast-components-react-msft/app/components/carousel-hero-content.tsx

```diff
@@ -2,21 +2,54 @@ import * as React from "react";
 import {
     CallToAction,
     CallToActionAppearance,
+    CallToActionProps,
     Heading,
+    HeadingProps,
     HeadingSize,
     HeadingTag,
     Image,
+    ImageProps,
     Paragraph,
+    ParagraphProps,
     ParagraphSize,
 } from "../../src/index";
 
+export interface CarouselHeroContentProps {
+    heading?: HeadingProps;
+    paragraph?: ParagraphProps;
+    callToAction: CallToActionProps;
+    image: ImageProps;
+    className?: string;
+}
+
 /**
  * A stand-in a mock hero
  */
-export default class CarouselHeroContent extends React.Component<{}, {}> {
+export default class CarouselHeroContent extends React.Component<
+    CarouselHeroContentProps,
+    {}
+> {
+    public static defaultProps: CarouselHeroContentProps = {
+        heading: {
+            children: "Heading text",
+        },
+        paragraph: {
+            children: "Hero paragraph test test",
```

Comment:
test test or test te**x**t

Reply:
Also, consider adding long text at times to capture overflow and localization issues.

Reply:
Normally yes, but this is a mock component (not a real hero in any way) so it doesn't have any constraints. I will fix the text though.